cli/script/command-executor.ts

```diff
@@ -896,11 +896,14 @@ function getReactNativeProjectAppVersion(command: cli.IReleaseReactCommand, proj
             throw new Error(`The "CFBundleShortVersionString" key doesn't exist within the "${resolvedPlistFile}" file.`);
         }
     } else if (command.platform === "android") {
-        const buildGradlePath: string = path.join("android", "app", "build.gradle");
+        let buildGradlePath: string = path.join("android", "app", "build.gradle");
```

Comment:
Well, in the case of the plistFile, it is actually a prefix (e.g. STAGING-Info.plist), but for the gradle file, since the name "build.gradle" is required, there's really no value to the user in typing it vs not typing it, so I would prefer if the same option could be smart enough to support both methods of specifying the path (the actual file path or just the containing folder).